### DISCUSSION TITLE
Multiple files at once

### DISCUSSION THREAD

**Discussion by Manniac07** [vectordotdev/vector#18597] [Q&A]
Hopefully this is something easy to fix. I am reading files off a directory and have tried using both Prometheus exporter and Prometheus remote write to send my metrics to Victoria metrics. Everything works good until I send 3 or more files at the same time. In this scenario, I am only able to see metrics from 2 files values come across and display in Grafana, I tried 6 files and only saw metrics for 3 files come across in Victoria Metrics.

**Comment by dsmith3197** [comment]
This looks to be a duplicate of https://github.com/vectordotdev/vector/discussions/18614. Please re-open if that is not the case.
